app.js

```diff
@@ -3,15 +3,17 @@ const app = express();
 const {
   getTopics,
   getArticleById,
+  patchArticleById,
 } = require("./db/controllers/news.controller");
 
 app.use(express.json());
 
 app.get("/api/topics", getTopics);
 app.get("/api/articles/:article_id", getArticleById);
+app.patch("/api/articles/:article_id", patchArticleById);
 
 app.use((err, req, res, next) => {
-  console.log(err.code);
+  console.log(err.code, "from app");
```

Comment:
Remove any console logs you've finished with please!

Reply:
Done!